channeldb/legacy_serialization.go

```diff
@@ -1,6 +1,13 @@
 package channeldb
 
-import "io"
+import (
```

Comment:
I really think this file should be renamed to something like `migration_09_legacy_serialization.go`. Just to make it very clear this is no longer used code.

Reply:
Done.